addons/website_sale/static/src/js/product/product.xml

```diff
@@ -9,21 +9,25 @@
         </QuantityButtons>
         <button name="sale_product_configurator_add_button" position="attributes">
             <attribute name="t-if">this.props.can_be_sold</attribute>
-            <attribute name="class" remove="btn-secondary" add="btn-primary" separator=" "/>
+            <attribute name="class" remove="btn-secondary" add="btn-outline-primary" separator=" "/>
         </button>
         <xpath
             expr="//button[@name='sale_product_configurator_add_button']/*[hasclass('fa-plus')]"
             position="attributes"
         >
             <attribute name="class" remove="fa-plus" add="fa-shopping-cart" separator=" "/>
         </xpath>
+        <xpath expr="//td[@name='price']/button[@name='sale_product_configurator_add_button']/span[contains(text(), 'Add')]" position="replace">
```

Comment:
Add name to that span `add_button` and add simple xpath here 
<span name="add_button"

and Do position attribute instead replacing element this we don't have to copy other style on the element and future adeptation